node_modules/titanium-sdk/lib/i18n.js

```diff
@@ -12,6 +12,13 @@ var appc = require('node-appc'),
 	DOMParser = require('xmldom').DOMParser;
 
 exports.load = function (projectDir, logger, opts) {
+	if(process.argv.indexOf('--i18n-directory')) {
+		// Enable developers to specify i18n directory location with build flag
+		var customI18n = process.argv[process.argv.indexOf('--i18n-directory')+1];
+		if(customI18n && fs.existsSync(path.join(path.resolve(projectDir), customI18n))) {
+			projectDir = path.join(projectDir, customI18n);
+		}
+	}
```

Comment:
Just being nitty-gritty. Can you format the above block ? For ex. spacing issues with if conditions - there should be a space between if and (